src/components/CreateAccount.js

```diff
@@ -3,12 +3,19 @@ import React, { Component } from 'react';
 import {
     INVALID_INFO,
     USERNAME_OR_EMAIL_TAKEN,
-    CREATE_ACCOUNT_FAILURE,
+    UNEXPECTED_ERROR,
     USERNAME_WARNING,
     PASSWORD_WARNING,
     EMAIL_WARNING,
 } from '../constants';
 import { Redirect } from 'react-router-dom';
+import {
+    Button,
+    PageHeader,
+    FormGroup,
+    Alert,
+    Label
+} from 'react-bootstrap';
```

Comment:
alphabetical imports please -> Alert, Button, FormGroup, etc.. 

also put react-bootstrap above react-router-dom

also put the relative files below the external packages

Ex. 
```
'.../constants'
<extra line here>
... from 'react-bootstrap';
```